### PR TITLE
Hmis 166

### PR DESCRIPTION
**Pull request by pats-john**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -112,16 +82,18 @@ stages:
...
+    - job: testsmoke
       pool:
         vmImage: 'windows-latest'
       steps:
       - task: Gradle@2
         inputs:
           gradleWrapperFile: 'gradlew'
           workingDirectory: '$(System.DefaultWorkingDirectory)'
-          tasks: clean smoke -DTEST_SUBSCRIPTION_KEY=$(hmi-apim-svc-sbox-sub-key) -DTEST_URL=https://hmi-apim-svc-sbox.azure-api.net -DTEST_HOST=hmi-apim-svc-sbox.azure-api.net
+          tasks: clean smoke -DTEST_SUBSCRIPTION_KEY=$(hmi-apim-sub-key) -DTEST_URL=https://hmi-apim-svc-sbox.azure-api.net -DTEST_HOST=hmi-apim-svc-sbox.azure-api.net
```
Comment (bugbare-reform) [inline azure-pipelines.yml:96]: shouldn't this one be test not sbox?

File: azure-pipelines.yml [hunk 2]
```diff
@@ -40,49 +28,31 @@ stages:
...
-          #     inlineScript: |
-          #       echo targetSubscriptionKey : \${TEST_SUBSCRIPTION_KEY:$(hmi-apim-svc-sbox-sub-key)} >> application-functional.yaml
-          #     workingDirectory: '$(System.DefaultWorkingDirectory)/src/functionalTest/resources/'
 
           - task: Gradle@2
             inputs:
               gradleWrapperFile: 'gradlew'
               workingDirectory: '$(System.DefaultWorkingDirectory)'
-              tasks: clean functional -DTEST_SUBSCRIPTION_KEY=$(hmi-apim-svc-sbox-sub-key) -DTEST_URL=https://hmi-apim-svc-sbox.azure-api.net -DTEST_HOST=hmi-apim-svc-sbox.azure-api.net
+              tasks: clean functional -DTEST_SUBSCRIPTION_KEY=$(hmi-apim-sub-key) -DTEST_URL=https://hmi-apim-svc-sbox.azure-api.net -DTEST_HOST=hmi-apim-svc-sbox.azure-api.net
```
Comment (bugbare-reform) [inline azure-pipelines.yml:55]: test not sbox?

File: script/read-vault.sh [hunk 3]
```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+username=$1
+password=$2
+subscriptionname=DTS-SHAREDSERVICES-SBOX
+vaultname=hmi-apim-kv-test
+secretname=hmi-apim-svc-test
+
+az login -u ${username} -p ${password}
```
Comment (bugbare-reform) [inline script/read-vault.sh:8]: are these variable names ok to use - if DevOps are ok with this then please ignore the query.

File: azure-pipelines.yml (hunk 1 above)
Comment (pats-john) [reply azure-pipelines.yml:96]: @bugbare-reform - This is not my implementation or file...It came up in the rebase from master....Not part of this implementation.

File: azure-pipelines.yml (hunk 2 above)
Comment (pats-john) [reply azure-pipelines.yml:55]: @bugbare-reform - This is not my implementation or file...It came up in the rebase from master....Not part of this implementation.

File: script/read-vault.sh (hunk 3 above)
Comment (pats-john) [reply script/read-vault.sh:8]: Will check with Dev ops...

**Comment by pats-john** [comment]
The issue is that we are having files that have a diff to master....I have to close and delete this branch....

**Pull request closed without merging**